geniza/corpus/templates/corpus/snippets/document_shelfmarks.html

```diff
@@ -1,10 +1,10 @@
 {% for block in document.textblock_set.all %}
     {% if block.certain %}
-        <span>{{ block.fragment.shelfmark }}</span>
+        <span>{{ block.fragment.shelfmark }}{% if not forloop.last %} + {% endif %}</span>
```

Comment:
This logic won't display correctly for uncertain joins if there are multiple or the uncertain join doesn't come last, but I just checked — there are only seven, they each only have two associated shelfmarks, and only one of them shows the problem (an extra +) because of the order of the certain / uncertain. I think we can live with that for now, until we address the uncertain join display problem properly.

Reply:
(alternative would be to make a `certain_shelfmarks` method that lists only those so the `forloop.last` is accurate)